docs/sphinx/theory/theory.rst

```diff
@@ -232,6 +232,65 @@ to *n+1/2*.
 Turbulence Models
 -----------------
 
+RANS models
+~~~~~~~~~~~~~
+
+The RANS models are available in two flavors: wall-modeled and wall-resolved. The former model is 
+designed for cases with :math:`y+ > 30` while the latter requires :math:`y+ < 5`. The wall-modeled RANS 
+model available in AMR-Wind is based on the work of `Axell and Liungman (EFM 2001 ) <https://link.springer.com/article/10.1023/A:1011560202388>`_.
+
+Axell One-Equation RANS Model 
```

Comment:
Can you add this one to the list here: https://github.com/Exawind/amr-wind/blob/main/docs/sphinx/user/features.rst?plain=1#L66 ?